### DISCUSSION TITLE
FluentButton height

### DISCUSSION THREAD

**Discussion by ivanjx** [microsoft/fluentui-blazor#581] [Q&A] [answered]
is there any way to set button height? i tried everything even manually editing the values via dev tools but no avail.

**Comment by vnbaaij** [comment]
You need to set the height on the actual button elements inside the shadow DOM of the web component. You can target that from CSS like this:
```
fluent-button::part(control) {
    height: 50px;
}
```

**Comment by ivanjx** [comment, marked as answer]
thanks for the answer but i found a better solution to just use `<FluentButton style="height: auto">`
